acceptance/README.md

```diff
@@ -48,12 +48,22 @@ source][beaker-hostgenerator].
 You'll need to provide a couple environment variables that specify which build
 of puppet-server to install and test against.
 
+##### Testing an existing build
 1. Go to http://builds.delivery.puppetlabs.net/puppetserver/
 2. Scroll down to the most recent build at the bottom.  This will look like:
-   `0.1.4.SNAPSHOT.2014.05.15T1118`
+   `2.2.2.master.SNAPSHOT.2016.02.18T1627`
 3. Copy the text (not the link address) - this will be `PACKAGE_BUILD_VERSION`
 4. Define ```PUPPET_VERSION``` as the packaged version of the puppet-agent
-   package that we're depending upon, which is currently `1.1.0`
+   package that we're depending upon, which is currently `1.3.5`
+
+##### Testing local changes
+1. run `lein with-profile ezbake ezbake build`. This will take a while, and
+   will result in a build appearing on http://builds.delivery.puppetlabs.net/puppetserver/
+2. If successful, it will give you back a URL with the new puppetserver version
+   at the end, which will look something like `2.2.2.master.SNAPSHOT.2016.02.18T1627`.
+   Set `PACKAGE_BUILD_VERSION` to this version.
+3. Define ```PUPPET_VERSION``` as the packaged version of the puppet-agent
+   package that your version of puppetserver depends upon
 
 If you are testing backwards compaitiblity against a specific release of
```

Comment:
It might be worth adding a note here stating that this applies whether or not you're testing local changes or an existing build